### PR TITLE
README.md: add debugging guide for Rancher Desktop users

### PR DESCRIPTION
**Pull request by AkihiroSuda**
Add debugging guide to help Rancher Desktop users:
```
LIMA_HOME="$HOME/Library/Application Support/rancher-desktop/lima" "/Applications/Rancher Desktop.app/Contents/Resources/resources/darwin/lima/bin/limactl" shell 0
```

Note that the path may differ in a future version of Rancher Desktop.

### CODE REVIEW

**Comment by jandubois** [comment]
On Linux:

```
LIMA_HOME="$HOME/.local/share/rancher-desktop/lima" /opt/rancher-desktop/resources/resources/linux/lima/bin/limactl shell 0
```

But I don't think this will work for the AppImage install; I don't know off-hand how this would work there, so maybe I will finally need to try the AppImage installer. 😄

**Comment by jandubois** [comment]
> But I don't think this will work for the AppImage install

No, it doesn't. AppImage installs into a temporary directory, and the name is different on each run. This command has been verified to work with the AppImage version:

```
LIMA_HOME="$HOME/.local/share/rancher-desktop/lima" "$(ls -d /tmp/.mount_ranche*/opt/rancher-desktop/resources/resources/linux/lima/bin)/limactl" shell 0
```

**Comment by AkihiroSuda** [comment]
@jandubois 
Thanks, updated PR to mention Linux version too

File: README.md [hunk 1]
```diff
@@ -368,6 +370,31 @@ To add another IP address that is accessible from the host and other virtual mac
...
+### External projects
+#### "I am using Rancher Desktop. How to deal with the underlying Lima?"
+
+On macOS hosts, Rancher Desktop (as of v1.0) launches Lima with the following configuration:
+
+- `$LIMA_HOME` directory: `$HOME/Library/Application Support/rancher-desktop/lima`
+- `limactl` binary: `/Applications/Rancher Desktop.app/Contents/Resources/resources/darwin/lima/bin/limactl`
+- Lima instance name: `0`
+
+e.g., To open a shell, run the following command:
```
Comment (jandubois) [inline README.md:382]:
```suggestion
To open a shell, run the following command:
```

**Review by jandubois — DISMISSED** [review]
Apart from the nitpicking, LGTM

**Review by jandubois — APPROVED** [review]

**Pull request merged**